package.json

```diff
@@ -67,10 +74,12 @@
     "@storybook/vue3": "~8.2.9",
     "@storybook/vue3-vite": "~8.2.9",
     "@vitejs/plugin-vue": "~5.1.2",
+    "@vue/test-utils": "^2.4.6",
     "autoprefixer": "~10.4.20",
     "eslint": "~9.9.0",
     "eslint-config-prettier": "~9.1.0",
     "globals": "~15.9.0",
+    "happy-dom": "^15.7.4",
```

Comment:
Question: why Happy DOM over JSDOM? JSDOM is what the other NeuRIS projects are using.

Reply:
No particular reason happy-do seemed to offer better speed. However, I swapped them out for consistency.